src/orchestration/queue.go

```diff
@@ -18,7 +15,7 @@ import (
 type QueueHandler struct {
 	queueClient           QueueClient
 	deadLetterQueueClient QueueClient
-	usecase               usecases.ReadAndSend
+	messageContentHandler MessageContentHandler
```

Comment:
Clients are interfaces and Handlers are structs naming wise. Is there a reason we're deviating?

Reply:
This is a good question! I think it's just coincidental that clients have been interfaces and handlers have been structs before (we do also have some interfaces and structs that are neither `clients` nor `handlers`, like in `ReadAndSend`). A lot of our `SomethingClient` interfaces have been created so we can mock third party clients (where a `client` is something that like...interacts with external sites/tools/resources), while the `SomethingHandler` structs associated with them have been about the logic of how we use/interact with the resources that the `SomethingClient` connects to. In this case, the `MessageContentHandler` is about the logic of what we do with the contents of a message, and it's an interface so that we can have a handler for import messages, a handler for polling messages, a handler for whatever-other-message-types-we-have-later, etc, that can all be used with the `QueueHandler`